### PR TITLE
disableAutoRotate added

### PR DESCRIPTION
**Pull request by AlexanderMS**
Supports this PR: https://github.com/Brightspace/lambda-key-management/pull/22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,15 @@ const keyGenerator = new KeyGenerator({
...
 ```
 
+#### Disabling auto-rotate
+
+If the module is used in a stateless context (such as, an AWS Lambda that cannot
+reliably hold the state in memory), default auto-rotation behaviour can be
+disabled by setting the option `disableAutoRotate: true`. In this case, the
+consumer is responsible for calling `generateNewKeys()` every `signingKeyAge`
+seconds and (if applicable) securely store the private key returned by the
+function. See [Lambda Key Management](https://github.com/Brightspace/lambda-key-management).
```
Comment (omsmith) [inline README.md]: You're linking to a private repo here

File: package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "check-style": "eslint .",
     "pretest": "npm run check-style",
-    "test": "istanbul cover -R _mocha -- --recursive ./test"
+    "test": "istanbul cover -R _mocha -- --recursive ./test",
+    "test-nocov": "mocha --recursive ./test"
```
Comment (omsmith) [inline package.json]: Is this because of stack traces?

Comment (AlexanderMS) [inline package.json]: It's that and also some issue with running tests on Windows does to some env var settings.

File: README.md (hunk 1 above)
Comment (AlexanderMS) [inline README.md]: True, need to remove it.

**Comment by AlexanderMS** [comment]
Yeah I thought of separating, but the idea I considered was leading me to a less clean and much bigger change that I would expect (and also breaking due to renaming):  renaming `KeyGenerator` to `RotatingKeyGenerator` and creating a separate `KeyGenerator` purely for generating keys. Both classes seemed to share almost all options passed to the constructors so it made less sense to me from the maintainability perspective. The short version: dunno either :D 
Any other thoughts?

**Comment by AlexanderMS** [comment]
https://github.com/Brightspace/node-auth-keys/pull/22

**Pull request closed without merging**